aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1606,6 +1779,9 @@ def _create_route_key(self, item: str, rule: str):
             )
         self._route_keys.append(route_key)
 
+    def _get_base_path(self) -> str:
+        raise NotImplementedError()
+
```

Comment:
Time to open a bug report in SAM?

Reply:
https://github.com/aws/aws-sam-cli/issues/5860

Reply:
Two things:

1. I can not move the code to the base class, because the `current_event` of the ApiGatewayResolver doesn't know about `request_context`, only its subclasses

2. I don't understand what your code is doing differently than mine. Care to explain with some example?